packages/fuels-test-helpers/src/node.rs

```diff
@@ -192,7 +192,8 @@ pub fn get_node_config_json(coins: Value) -> NamedTempFile {
         "max_static_contracts": 255,
         "max_storage_slots": 255,
         "max_predicate_length": 1048576,
-        "max_predicate_data_length": 1048576
+        "max_predicate_data_length": 1048576,
+        "gas_price_factor": 1000000000
```

Comment:
Does this allow trailing comma?
```suggestion
        "gas_price_factor": 1000000000,
```